cypress/integration/admin_portal/tests/read_email_template.js

```diff
@@ -0,0 +1,15 @@
+import HelperFunctions from '../helpers/helper_functions'
+
+describe('Email assertion:', () => {
+  const helpers = new HelperFunctions()
+  it('Gets the email on basis of a search query', function () {
+    const mailOptions = {
+      from: 'activate@edx.org',
+      to: helpers.getUniqueEmailAlias(),
+      subject: 'Activate',
+    }
+    cy.task('mailReader', mailOptions).then((email) => {
+      cy.log(helpers.extractUrlFromEmail(email, 'activate'))
```

Comment:
Don't we need to assert/verify anything in this test?

Reply:
it's just a template to show email reading is working, actual tests will be added afterwards